pyoverkiz/clients/somfy_local.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from typing import Any
+
+from pyoverkiz.clients.overkiz import OverkizClient
+from pyoverkiz.types import JSON
+
+
+class SomfyLocalClient(OverkizClient):
+    async def _login(self, username: str, password: str) -> bool:
+        """There is no login for Somfy Local"""
+        return True
+
+    @property
+    def _headers(self) -> dict[str, str]:
+        return {"Authorization": f"Bearer {self.password}"}
+
+    async def get(
+        self,
+        path: str,
+        _ssl: bool = False,
+    ) -> Any:
+        """Make a GET request to the OverKiz API"""
+
+        return await super().get(path, ssl=False)
+
+    async def post(
+        self,
+        path: str,
+        payload: JSON | None = None,
+        data: JSON | None = None,
+        _ssl: bool = False,
```

Comment:
Should we set the SSL=False not in the session? Isn't this something we should not define hardcoded in our get() implementation.

Reply:
Yes indeed, I forgot to push my latest commits.